crates/rooch-framework/sources/ord.move

```diff
@@ -42,5 +71,54 @@ module rooch_framework::ord {
         from_transaction(&transaction)
     }
 
-    public native fun from_witness(witness: &Witness): vector<Inscription>;
+    native fun from_witness(witness: &Witness): vector<Inscription>;
+
+    public fun total_inscriptions(inscription_store_obj: &Object<InscriptionStore>): u64{
+        let inscription_store = object::borrow(inscription_store_obj);
+        table_vec::length(&inscription_store.inscription_ids)
+    }
+
+    public fun remaining_tx_count(btc_block_store_obj: &Object<BitcoinBlockStore>, inscription_store_obj: &Object<InscriptionStore>): u64{
+        let inscription_store = object::borrow(inscription_store_obj);
+        let start_tx_index = inscription_store.next_tx_index;
+        let max_tx_count = table_vec::length(bitcoin_light_client::tx_ids(btc_block_store_obj));
+        if(start_tx_index < max_tx_count){
+            max_tx_count - start_tx_index
+        }else{
+            0
+        }
+    }
+
+    entry fun progress_inscriptions(btc_block_store_obj:&Object<BitcoinBlockStore>, inscription_store_obj: &mut Object<InscriptionStore>,batch_size: u64){
+        let inscription_store = object::borrow_mut(inscription_store_obj);
+        let txs:&Table<address, Transaction> = bitcoin_light_client::txs(btc_block_store_obj);
+        let tx_ids: &TableVec<address> = bitcoin_light_client::tx_ids(btc_block_store_obj);
+        let start_tx_index = inscription_store.next_tx_index;
+        let max_tx_count = table_vec::length(tx_ids);
+        if(start_tx_index >= max_tx_count){
+            return
+        };
+        let progressed_tx_count = 0;
```

Comment:
Shouldn't there be renamed to `processed_tx_count`?

Reply:
make a PR